libraries/freertos_plus/standard/freertos_plus_tcp/source/FreeRTOS_DNS.c

```diff
@@ -1286,6 +1287,7 @@ uint16_t usType = 0U;
 							if( xDoStore != pdFALSE )
 							{
 								( void ) prvProcessDNSCache( pcName, &ulIPAddress, pxDNSAnswerRecord->ulTTL, pdFALSE );
+                                usNumARecordsStored++;    /* Track # of A records stored */
```

Comment:
Please indent this to be aligned with scope

Reply:
This file uses tabs unfortunately. I have updated my changes to use tabs rather than spaces.

Reply:
> This file uses tabs unfortunately

Why is that unfortunately? All FreeRTOS sources use tabs with a width of 4.

The white space was "correct" as it was:
~~~c
for( x = 0U; ( x < pxDNSMessageHeader->usAnswers ) && ( usNumARecordsStored < usCount ); x++ )
{
BaseType_t xDoAccept;

	uxResult = prvSkipNameField( pucByte, uxSourceBytesRemaining );
~~~
FreeRTOS style: local declarations in a function do not indent. See e.g. in tasks.c :
~~~c
void vTaskDelete( TaskHandle_t xTaskToDelete )
{
TCB_t *pxTCB;

    taskENTER_CRITICAL();
~~~